src/ResourceManager/Profile/Commands.Profile/Az.Profile.psd1

```diff
@@ -54,19 +54,19 @@ PowerShellVersion = '5.1'
 # RequiredModules = @()
 
 # Assemblies that must be loaded prior to importing this module
-RequiredAssemblies = '.\Microsoft.Azure.Commands.Common.Authentication.Abstractions.dll', 
-    '.\Microsoft.Azure.Commands.Common.Authentication.dll', 
-    '.\Microsoft.Azure.Commands.Common.Authentication.ResourceManager.dll', 
-    '.\Microsoft.Azure.Commands.Common.Authorization.dll', 
-    '.\Microsoft.Azure.Commands.Common.Compute.dll', 
-    '.\Microsoft.Azure.Commands.Common.Graph.RBAC.dll', 
-    '.\Microsoft.Azure.Commands.Common.Network.dll', 
-    '.\Microsoft.Azure.Commands.ResourceManager.Common.dll', 
-    '.\Microsoft.WindowsAzure.Commands.Common.dll', 
-    '.\Microsoft.WindowsAzure.Commands.Common.Storage.dll', 
-    '.\Microsoft.Azure.Commands.Common.KeyVault.dll', 
-    '.\Microsoft.Azure.Commands.Common.Storage.Management.dll', 
-    '.\Microsoft.Azure.Commands.Common.Websites.dll', 
+RequiredAssemblies = '.\Microsoft.Azure.PowerShell.Authentication.Abstractions.dll', 
+    '.\Microsoft.Azure.PowerShell.Authentication.dll', 
+    '.\Microsoft.Azure.PowerShell.Authentication.ResourceManager.dll', 
+    '.\Microsoft.Azure.PowerShell.Authorization.dll', 
+    '.\Microsoft.Azure.PowerShell.Compute.dll', 
+    '.\Microsoft.Azure.PowerShell.Graph.Rbac.dll', 
+    '.\Microsoft.Azure.PowerShell.Network.dll', 
```

Comment:
these will need to change back

Reply:
These will not change because this is the NetStandard psd1 for Profile.